### PR TITLE
Appneta Annotations

### PR DESCRIPTION
**Pull request by nbommu1**
##### ISSUE TYPE

- New Module Pull Request
##### COMPONENT NAME

appneta_annotations
##### ANSIBLE VERSION

```
1.9.4
```
##### SUMMARY

appneta_annotations - Notify appneta about app deployments

### CODE REVIEW

**Comment by gregdek** [comment]
Thanks @nbommu1 for this PR. This PR requires revisions, either because it fails to build or by reviewer request. Please make the suggested revisions. When you are done, please comment with text 'ready_for_review' and we will put this PR back into review.

[This message brought to you by your friendly Ansibull-bot.]

**Comment by nbommu1** [comment]
ready_for_review

File: monitoring/appneta_annotations.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+"""
+DOCUMENTATION = '''
+
+module: appneta_annotations
+version_added: "1.0"
```
Comment (gundalow) [inline monitoring/appneta_annotations.py]: This should be `2.2`

File: monitoring/appneta_annotations.py [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+            appname=dict(required=False),
+            hostname=dict(required=False),
+            username=dict(required=False),
+        ),
+        supports_check_mode=True
+    )
+
+    # build list of params
+    params = {}
+    if module.params["appname"]:
```
Comment (gundalow) [inline monitoring/appneta_annotations.py:88]: I might not be reading this correctly, but should `appname` be a required paramater

e.g.

`appname=dict(required=True),`

And update the docs

**Comment by gundalow** [comment]
Hi,

Looks like there is a small tweak needed to support Python 3

```
python3 -m compileall -fq yourmodule.py 
```

The following guide should explain what's needed http://docs.ansible.com/ansible/developing_modules_python3.html

needs_revision

Thanks

**Comment by nbommu1** [comment]
ready_for_review

**Comment by mattclay** [comment]
Tests on shippable failed because this PR branch was created before shippable was enabled. If you rebase the branch the tests will pass, although in this case the tests will just no-op since there are no tests for this module.

Tests also failed on travis. Rebasing should resolve that issue as well.

**Comment by nbommu1** [comment]
ready_for_review

**Comment by gregdek** [comment]
Thanks @nbommu1 for this new module. When this module receives 'shipit' comments from two community members and any 'needs_revision' comments have been resolved, we will mark for inclusion.

[This message brought to you by your friendly Ansibull-bot.]

**Pull request closed without merging**